### PR TITLE
feat(workspace*) Wks color selection is now the same as queryOverlayStyle.selection config

### PR DESCRIPTION
**Pull request by josee666**
**What is the current behavior?** (You can also link to an open issue here)
https://github.com/infra-geo-ouverte/igo2/issues/765

**What is the new behavior?**
The color selection is now the same as config queryOverlayStyle.selection
        "fillColor" 
        "fillOpacity"
        "strokeColor"
        "strokeWidth"      

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [x ] No

**Other information**:
config to test:
```
 "queryOverlayStyle": {

    "selection": {
        "markerColor": [0,137,123], 
        "markerOpacity": 1.0, 
        "markerOutlineColor": [255,255,255],        
        "fillColor": "#c9f2d0", 
        "fillOpacity": 0.0,     
        "strokeColor": "#2effff",       
        "strokeWidth": 3                
    },
```

layerContext to test:
  {     
        "title": "WMS-WFS avec table attribut sans query",
        "visible": true,
        "id": 1,
        "workspace": {
          "enabled": true,
          "maxResolution": 300,
          "queryOptions": {
            "mapQueryOnOpenTab": false,
            "tabQuery": false
          }
        },
        "sourceOptions": {
          "queryable": true,
          "queryTitle": "photographies aériennes",
          "crossOrigin": "anonymous",
          "queryFormat": "gml2",
          "queryHtmlTarget": "iframe",
          "queryFormatAsWms": true,
          "type": "wms",
          "optionsFromCapabilities": true,
          "url": "/ws/mffpecofor.fcgi",
          "params": {
            "layers": "nord_photo_oblique",
            "version": "1.3.0"
          },
          "urlWfs": "/ws/mffpecofor.fcgi",
          "paramsWFS": {
            "featureTypes": "nord_photo_oblique",
            "maxFeatures": 1000,
            "version": "2.0.0"
          },
          "sourceFields": [
            {"name": "LATITUDE", "alias": "latitude"},
            {"name": "LONGITUDE", "alias": "longitude"},
            {"name": "NOM_PHOTO", "alias": "nom photo"} 
          ] 
        }
      },
      {
        "title": "Vector geojson d'appel WFS",
        "id": "vector2",
        "workspace": {
          "enabled": true,
          "queryOptions": {
              "mapQueryOnOpenTab": false,
              "tabQuery": false
          }
      },
        "sourceOptions": {
          "type": "vector",
          "url": "https://ws.mapserver.transports.gouv.qc.ca/swtq?service=WFS&request=GetFeature&version=1.1.0&typename=aeroport_piste&outputFormat=geojson",
          "queryable": true,
          "queryFormat": "geojson",
          "queryTitle": "nomnavcana"
        }
      },
    {
      "title": "feux tab + filtre",
      "id": "test",
      "visible": true,
      "workspace": {
          "enabled": true,
          "maxResolution": 300,
          "queryOptions": {
              "mapQueryOnOpenTab": false,
              "tabQuery": false
          }
      },
      "metadata": {
      "extern": true
      },
      "sourceOptions": {
        "crossOrigin": "anonymous",
        "queryable": true,
        "queryFormat": "htmlgml2",
        "queryHtmlTarget": "iframe",
            "queryFormatAsWms": true,
            "type": "wms",
            "optionsFromCapabilities": true,
            "url": "/ws/mffpecofor.fcgi", 
            "params": {
                "layers": "ca_feux_close_scale",
                "version": "1.3.0"
            },
            "urlWfs": "/ws/mffpecofor.fcgi",
            "paramsWFS": {
              "featureTypes": "ca_feux_close_scale",
              "fieldNameGeometry": "the_geom",
              "maxFeatures": 3000,
              "version": "2.0.0"
            },
           
            "timeFilterable": false,
            "ogcFilters": {
                "enabled": true,
                "editable": true,
                "allowedOperatorsType": "basic",
                "pushButtons": {
                  "groups": [
                    {"title": "Group 1 Title","name": "1","ids": ["id1"]}
                  ],
                  "bundles": [
                    {
                      "id": "id1",
                      "logical": "Or",
                      "title": "Type feu",
                      "selectors": [
                      {
                          "title": "total",
                          "color": "0,137,123",
                          "filters": {
                              "operator": "PropertyIsEqualTo",
                              "propertyName": "symbologie",
                              "expression": "BR"
                          }
                      },
                        {
                          "title": "partiel",
                          "color": "0,137,123",
                          "filters": {
                            "operator": "PropertyIsEqualTo",
                            "propertyName": "symbologie",
                            "expression": "BRP"
                          }
                        }
                      ]
                    }
                  ]
                  },
                "filters":
                {
                  "operator": "During",
                  "propertyName": "annee_date",
                  "begin": "1890-01-01",
                  "end": "2021-01-01",
                  "calendarModeYear": true,
                  "restrictToStep": false
                } 
            },
            "stepDate": "P1Y",
            "minDate": "1890-01-01",
            "maxDate": "2022-01-01"

        }       
      }

### CODE REVIEW

File: packages/geo/src/lib/workspace/shared/feature-workspace.service.ts [hunk 1]
```diff
@@ -81,11 +83,36 @@ export class FeatureWorkspaceService {
     const inMapExtentStrategy = new FeatureStoreInMapExtentStrategy({});
     const inMapResolutionStrategy = new FeatureStoreInMapResolutionStrategy({});
     const selectedRecordStrategy = new EntityStoreFilterSelectionStrategy({});
+    let styles = undefined;
```
Comment (pelord) [inline packages/geo/src/lib/workspace/shared/feature-workspace.service.ts:86]: why not handle the selection style the same way as others?

getCommonVectorSelectedStyle or getCommonVectorStyle....

File: packages/geo/src/lib/workspace/shared/wfs-workspace.service.ts [hunk 2]
```diff
@@ -78,11 +80,35 @@ export class WfsWorkspaceService {
     const inMapExtentStrategy = new FeatureStoreInMapExtentStrategy({});
     const inMapResolutionStrategy = new FeatureStoreInMapResolutionStrategy({});
     const selectedRecordStrategy = new EntityStoreFilterSelectionStrategy({});
+    let styles = undefined;
```
Comment (pelord) [inline packages/geo/src/lib/workspace/shared/wfs-workspace.service.ts:83]: same comment as previous. ....

File: packages/geo/src/lib/workspace/shared/wms-workspace.service.ts [hunk 3]
```diff
@@ -179,11 +181,35 @@ export class WmsWorkspaceService {
     const inMapExtentStrategy = new FeatureStoreInMapExtentStrategy({});
     const inMapResolutionStrategy = new FeatureStoreInMapResolutionStrategy({});
     const selectedRecordStrategy = new EntityStoreFilterSelectionStrategy({});
+    let styles = undefined;
```
Comment (pelord) [inline packages/geo/src/lib/workspace/shared/wms-workspace.service.ts:184]: same comment.

**Review by pelord — CHANGES_REQUESTED** [review]
Modify as requested based on common methods already used in the code.

File: packages/geo/src/lib/workspace/shared/feature-workspace.service.ts (hunk 1 above)
Comment (josee666) [reply packages/geo/src/lib/workspace/shared/feature-workspace.service.ts]: Change made and get style from getCommonVector. Thanks for comment

**Review by pelord — APPROVED** [review]

**Comment by pelord** [comment]
@PhilippeLafreniere18 Do you take a look?

**Review by PhilippeLafreniere18 — CHANGES_REQUESTED** [review]
Need to add "outlineColor" to CommonVectorStyleOptions interface ? Except that, works fine for me.

**Comment by pelord** [comment]
> 

It's already handled by strokeColor 
https://github.com/infra-geo-ouverte/igo2-lib/blob/1.11.1/packages/geo/src/lib/utils/commonVectorStyle.interface.ts

**Review by PhilippeLafreniere18 — APPROVED** [review]

**Pull request merged**